### PR TITLE
Add validation for pattern files

### PR DESCRIPTION
**Pull request by MUzairS15**
Signed-off-by: MUzairS15 <muzair.shaikh810@gmail.com>

**Description**

This PR adds validation for pattern files in using `/validate` endpoint.
Screenshot 2022-09-24 at 9 48 56 PM

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@sudo-NithishKarthik this uses your new endpoint, right? A review and comment from you would be great.

**Review by Chadha93 — DISMISSED** [review]

**Comment by Chadha93** [comment]
LGTM, let's get a review from @sudo-NithishKarthik and @Abhishek-kumar09 :+1:

File: ui/components/MesheryPatterns.js [hunk 1]
```diff
@@ -483,6 +503,60 @@ function MesheryPatterns({
     );
   };
 
+  const handleVerify = (e, pattern_file, pattern_id) => {
```
Comment (sudo-NithishKarthik) [inline ui/components/MesheryPatterns.js:506]: @MUzairS15 , I believe this is same as the one that I have written in MeshMap. In that case, let us just keep one copy of this and reuse it. There should be a function that verifies a pattern file using the new validation endpoint and it should be reused by both Meshery UI and MeshMap

File: ui/components/Validation.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  error : {
+    position : "relative",
+    left : "-50px",
+    borderRadius : "0.4rem",
+    padding : "0.5rem",
+    top : "-0.45rem"
+  }
+});
+
+const ValidationComponent = ({ errors, compCount, classes, handleClose }) => {
```
Comment (sudo-NithishKarthik) [inline ui/components/Validation.js:40]: Is this the same as the one used in MeshMap?

**Comment by MUzairS15** [comment]
https://github.com/layer5labs/meshery-extensions/pull/1229
This is the (WIP) PR by @Chadha93 for the changes.

**Comment by leecalcote** [comment]
@MUzairS15, this PR has no dependency on the WIP by, @Chadha93, right? (likely, the inverse is true)

**Review by leecalcote — DISMISSED** [review]

**Comment by MUzairS15** [comment]
No dependency, but @Chadha93 PR depends on this.

**Review by leecalcote — APPROVED** [review]
Unless I've missed something, I think we're good to merge here, @MUzairS15

**Comment by MUzairS15** [comment]
Yes good to go.

**Pull request merged**